src/components/table-of-contents/data.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
Test data is usually called a fixture, so that it's clear it's testing related. So `fixtures.json`?

Reply:
done